Gemfile

```
@@ -12,6 +12,8 @@ gem 'dotenv-rails', '~> 2.7', '>= 2.7.6' # https://rubygems.org/gems/dotenv-rail
 gem 'jbuilder', '~> 2.7' # https://rubygems.org/gems/jbuilder
 gem 'pg', '~> 1.1' # https://rubygems.org/gems/pg
 gem 'puma', '~> 5.0' # https://rubygems.org/gems/puma
+gem 'discard', '~> 1.2'
```

Esta gema la podías ubicar en la linea 11 (debajo de bootsnap) para respetar el orden alfabético. En el formato sugerido:
```suggestion
gem 'discard', '~> 1.2' # https://rubygems.org /gems/discard
```